### ISSUE TITLE
Confusion About "--output-names "a/F.sigmoid(d0)"

### ISSUE THREAD

**Issue by NaeemKhan333** [w-okada/image-analyze-workers#24]
@w-okada Thanks for such a nice web demos . I know this is not repository for model conversion but I am facing problem in conversion , So I am asking this question here

* Question related to u2net portrait conversion 
 
 As in your [Blog](https://medium.com/analytics-vidhya/generate-portrait-on-browser-with-u%C2%B2-net-71b4c682df44), you have converted u2net-portrait model into tensorflow Js model and Once the conversion is complete, you place it on your website and load it with Tensorflowjs for use.
 
 For conversion of pytorch model into onnx , you have run the following command 
 
 ```
 python3 /opt/intel/openvino_2021/deployment_tools/tools/model_downloader/pytorch_to_onnx.py  \
 --import-module model.u2net   \
 --model-name U2NET   \
 --input-shape 1,3,${SIZE},${SIZE}   \
 --weights saved_models/u2net_portrait/u2net_portrait.pth   \
 --output-file u2netp_${SIZE}x${SIZE}.onnx   --input-names "x"   \
 --output-names "a/F.sigmoid(d0)"
 ```
 I have confusion about "--output-names "a/F.sigmoid(d0)" , what is actully means and what is purpose of the of this . As per I understand, you are exporting/getting the output of d0 variable because our mask is relies inside this variable . If my understanding is not right , can you give elaboration of this ""a/F.sigmoid(d0)"".
 
Secondly while I have converted u2net-portrait model using your blog . While I run the converted tensorflow js model, It is giving me 7(d0....d6) outputs, I do not know why it is giving me 7 outputs, actully we export d0 output . Can you recheck it may be there is problem in the command due to update in the u2net-model or openvino version . It is highly appreciable if you response to my query. Thanks

**Comment by NaeemKhan333** [comment]
I have used [Neutron](https://github.com/lutzroeder/netron) software to compare your converted u2net-portrait Model with myself converted u2net-portrait model.I am uploading the output shape of your converted model and myself converted model . 

Your u2net-portrait-Converted Model:

![your-blog-model](https://user-images.githubusercontent.com/52487215/107371288-f1c28e00-6b05-11eb-9b46-2c7e43a79708.png)

My u2net-portrait-Converted Model:

![Screenshot from 2021-02-09 18-36-08](https://user-images.githubusercontent.com/52487215/107371353-043cc780-6b06-11eb-8cf7-e6b48b11adb0.png)

There is difference of my converted model with your converted u2net portrait model. Can you recheck,whether there is update in model or any other update which are effecting the model conversion. Can you guide me about it . Thanks

**Comment by w-okada** [comment]
I don't know why you got the diffrent network from mine. It is not strange the tools output diffrent value when the tools are updated. I don't have detail history about which commit, which model or so. What I described in my blog is all I can provide.

By the way, what is problem the network is different from mine? I cannot understand what you need. If you want to match the result to mine exactly, I think it is very hard. And there is no value to do so.

**Comment by w-okada** [comment]
no response. close
